.env.example

```diff
@@ -335,8 +335,6 @@ AB_TEST_BUCKET_PROBABILITY=50
 APP_ROUTER_EVENT_TYPES_ENABLED=0
 APP_ROUTER_SETTINGS_DEVELOPER_ENABLED=0
 APP_ROUTER_SETTINGS_ADMIN_ENABLED=0
-APP_ROUTER_SETTINGS_MY_ACCOUNT_ENABLED=0
```

Comment:
If you wonder where `APP_ROUTER_SETTINGS_LICENSE_KEY_ENABLED` is, we do not have it :P